test/test_faiss_and_milvus.py

```diff
@@ -90,6 +94,10 @@ def test_faiss_index_save_and_load_custom_path(tmp_path):
     # Check if the init parameters are kept
     assert not new_document_store.progress_bar
 
+    # test saving and loading the loaded faiss index
+    new_document_store.save(tmp_path / "haystack_test_faiss", config_path=tmp_path / "custom_path.json")
+    new_document_store.load(tmp_path / "haystack_test_faiss", config_path=tmp_path / "custom_path.json")
```

Comment:
That should be `FAISSDocumentStore.load(...)` instead and we should store the return value in a variable with a different name, e.g., `reloaded_document_store`.

Let's add the assertions afterwards:
```
# check faiss index is restored
assert reloaded_document_store.faiss_indexes[document_store.index].ntotal == len(DOCUMENTS)
# check if documents are restored
assert len(reloaded_document_store.get_all_documents()) == len(DOCUMENTS)
# Check if the init parameters are kept
assert not reloaded_document_store.progress_bar
```

Reply:
Done! :)